common.c

```diff
@@ -35,40 +36,104 @@ struct addrinfo *get_address(const char *host, const char *port)
     }
 }
 
+#ifdef __linux__
+    #ifndef UDP_SEGMENT
+        #define UDP_SEGMENT 103
+    #endif
+#endif
```

Comment:
Since this is a global definition, it should be moved to the top. Maybe add a comment why this is only defined on linux

Reply:
See comment above regarding consistency to quicly. In https://github.com/rbruenig/qperf/pull/5/commits/1d27abd4b19a334ddd4a9b2e58def0011656dcc3 I changed the `#ifndef` to include the linux-only `send_dgrams_gso` function.